### PR TITLE
Add custom fonts

### PR DESCRIPTION
**Pull request by pirsquareff**

### CODE REVIEW

File: frontend-webapp/src/styles/_helper.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/* ==========================================================================
+    Typography
+   ========================================================================== */
+
+._font-heading {
```
[the remark this answers is not in the document]
Comment (pirsquareff) [reply frontend-webapp/src/styles/_helper.scss]: Done.

File: frontend-webapp/src/app/shared/input/input-group.component.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+@import '~styles/common';
+@import '~styles/helper';
```
Comment (futurizing) [inline frontend-webapp/src/app/shared/input/input-group.component.scss]: Can we merge common and helper into a single file? Also, for small file size, the common file should only not have side-effect classes, e.g. contains only mixins.

File: frontend-webapp/src/app/shared/input/input-group.component.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+label {
```
Comment (futurizing) [inline frontend-webapp/src/app/shared/input/input-group.component.scss]: Should this be in the InputGroupComponent?

File: frontend-webapp/src/app/shared/input/input-group.component.scss [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+  font-family: $headings-font-family;
+  @extend ._font-regular;
```
Comment (futurizing) [inline frontend-webapp/src/app/shared/input/input-group.component.scss]: If possible, extending class is discourage in non-global scss file, as it indicates that there are imported classes which might be unnecessary (see upper comment).

File: frontend-webapp/src/styles/_common.scss [hunk 5]
```diff
@@ -17,6 +17,8 @@ $border-radius: .125rem;
 $border-radius-lg: .15rem;
 $border-radius: .1rem;
 
+$headings-font-family: 'Lato', 'Mitr', sans-serif !important;
```
Comment (futurizing) [inline frontend-webapp/src/styles/_common.scss]: Why !important?

File: frontend-webapp/src/styles/_helper.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (futurizing) [reply frontend-webapp/src/styles/_helper.scss]: I mean completely remove _font-heading class and make it as a mixin instead.

File: frontend-webapp/src/styles/global.scss [hunk 6]
```diff
@@ -38,3 +40,29 @@
 @import '~bootstrap/scss/popover';
 // @import '~bootstrap/scss/carousel';
 @import '~bootstrap/scss/utilities';
+
+html {
+  font-family: $font-family-sans-serif;
```
Comment (futurizing) [inline frontend-webapp/src/styles/global.scss]: Should not be needed.

File: frontend-webapp/src/styles/global.scss [hunk 7]
```diff
@@ -38,3 +40,29 @@
...
+  @extend ._font-regular;
+  font-size: 15px;
```
Comment (futurizing) [inline frontend-webapp/src/styles/global.scss]: Override `$font-size-base` instead.

File: frontend-webapp/src/styles/global.scss [hunk 8]
```diff
@@ -38,3 +40,29 @@
...
+}
+
+h1, h2, h3 {
+  font-family: $headings-font-family;
```
Comment (futurizing) [inline frontend-webapp/src/styles/global.scss]: Should not be needed.

File: frontend-webapp/src/styles/global.scss [hunk 9]
```diff
@@ -38,3 +40,29 @@
...
+  @extend ._font-bold;
```
Comment (futurizing) [inline frontend-webapp/src/styles/global.scss]: Override `$headings-font-weight` instead.

File: frontend-webapp/src/styles/global.scss [hunk 10]
```diff
@@ -38,3 +40,29 @@
...
+}
+
+.form-control-lg {
+  padding: 0.8rem 1rem;
+  font-size: 1rem;
+}
+
+.btn-lg {
+  padding: 0.8rem 1.2rem;
```
Comment (futurizing) [inline frontend-webapp/src/styles/global.scss]: Use `$input-btn-padding-y-lg` and `$input-btn-padding-x-lg` instead.

File: frontend-webapp/src/app/feed/feed.component.html [hunk 11]
```diff
@@ -5,12 +5,12 @@
     <form [formGroup]="searchForm" class="row align-items-end">
       <div class="col-12 col-lg-5">
         <app-input-group label="ชื่อตลาดหรือสถานที่">
-          <input appInput type="text" class="form-control form-control-lg" placeholder="ชื่อตลาดหรือสถานที่">
+          <input appInput type="text" class="form-control btn-lg" placeholder="ชื่อตลาดหรือสถานที่">
```
Comment (futurizing) [inline frontend-webapp/src/app/feed/feed.component.html]: I think `form-control-lg` is already correct. :)

**Comment by pirsquareff** [comment]
Try fixing as requested, please check again.
@futurizing

File: frontend-webapp/src/app/feed/feed.component.html (hunk 11 above)
Comment (pirsquareff) [reply frontend-webapp/src/app/feed/feed.component.html]: Okay. I will change back.

**Pull request merged**